### ISSUE TITLE
It not showing wrap_content

### ISSUE THREAD

**Issue by cchan20** [rameshvoltella/RWeekCalendar#2]
I have set the layout_height as wrap_content,, but it is still display as full screen. Even I try to set it as a constant height as 100dp, it still showing as full screen like fill_parents.

**Comment by rameshvoltella** [comment]
how you design in xml? can you show the code?

**Comment by cchan20** [comment]
<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    android:orientation="vertical">



```
<LinearLayout
    android:id="@+id/calendar"
    android:layout_width="match_parent"
    android:layout_height="wrap_content"
    android:orientation="vertical">
</LinearLayout>
```

</RelativeLayout>

This is my calendar view xml,, and I have tried to put 150dp on android:layout_height,, but it still shows as full screen.

**Comment by rameshvoltella** [comment]
can u send your screen shot?

**Comment by cchan20** [comment]
I just found out the solution. RWeekCalendar is a fragment, and I add this into my fragment page, so this case is using nested fragments. If using the getSupportFragmentManager(), it would replace my whole fragment page,  but if I change it to getChildFragmentManager(), it will just replace the specific layout.

Thanks man^^

Btw, I found that this library is not supporting the date before current date. If I select date after current date, it's fine. But if I select the date before current date, it doesn't work.
